jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -54,6 +57,7 @@
         .with("file:")
         .with("jrt:")
         .with("jar:")
+        .with("resource:")
```

Comment:
Nope, this should not be part of the default schemes.
This scheme has dangerous side effects outside of GraalVM and should be only be allowed to be configured, not always on.

Reply:
Is there another environment where this scheme is actually available?
What specific dangerous side effects do you have in mind?

I think `resource:` URLs/URIs are exclusive to GraalVM native-image. On a regular hotspot VM, I simply get an error when `resource:` URLs are requested.

Besides that, URIUtil.KNOWN_SCHEMES is actually not used and private.

Reply:
URIUtil.KNOWN_SCHEMES is actually not used, and private. Can we remove the entire declaration?

Reply:
if it's unused, i don't see why not.